### DISCUSSION TITLE
xDrip+ 2022.04.20 won't calibrate

### DISCUSSION THREAD

**Discussion by CLKRUN** [NightscoutFoundation/xDrip#2084] [Q&A]
Hello,

I'm using xDrip+ (2022.04.20) on Pixel 6 with Android 12. My data source is Dex Share Follow.
For example my Dexcom App shows 160 mg/dl and in xDrip I did a inital calibration of 207 mg/dl.
After the calibration the graph stayed at 207 but if the next value from Dex Share Follow coming it overrides to the "wrong" value.

Is it a bug? data source crap? Android 12? Pixel 6? 
Attached a screenshot of the log (cant upload here, <<Something went really wrong, and we can’t process that file.>>)
https://imgur.com/a/oBfuL7K

**Comment by Navid200** [comment]
So, you are using xDrip as a follower.
And, want to calibrate from the follower, not the master?
Is the device a G6 or G5?

@tolot27 Can a dex share follower calibrate?

**Reply by CLKRUN** [reply]
I'm getting the data from Dexcom Share. It's a 2nd phone (iphone se) who's using official g6 app and is connected to my transmitter.
yes, correct. The Pixel 6 is not the master.
G6. I think with  8G*** transmitters.

**Reply by Navid200** [reply]
Why can you not calibrate from the official G6 app?

**Reply by CLKRUN** [reply]
if i do, why should i still use xdrip? ;-)
I would like to test whether xdrip is after calibration as well as insulin injections to the point with bloody values. As far as I know, this does not work with the official Dexcom app. I had at the very beginning of my disease Libre + xDrip and it was super close to the bloody values. Besides, I don't feel like carrying around two smartphones anymore.

**Reply by Navid200** [reply]
Something isn't clear here.
I assume you are the diabetic.  
You are using the Dexcom app as the collector.  So, you have to carry the phone that has the Dexcom app on it.
But, you like to use xDrip as a follower.   So, you will have to carry the phone with xDrip on it also.

Which one of those assumptions is incorrect?

**Reply by CLKRUN** [reply]
Yes, that's true. But now I want to test xDrip, because as long as Dexcom doesn't support Android 12 and Google Pixel 6 I have to use both phones. Especially since xDrip offers more features than Dexcom.
I could also run a Dexcom patched app on the Pixel 6, but I'm still undecided.
You do understand that at some point I want to get to the point where I only want to carry one mobile device?
That's why my approach so complicated.

**Reply by Navid200** [reply]
You cannot calibrate xDrip.
You need to calibrate the G6 transmitter.

I'm not sure a Dex share follower can submit a calibration to the transmitter.
Hopefully, someone who has experience will help you.

Good luck.

**Reply by CLKRUN** [reply]
Thanks for your support. Would be great to hear some feedback.

**Reply by tolot27** [reply]
> @tolot27 Can a dex share follower calibrate?

No, calibration happens only locally at the collector side because it sends the bg value to the transmitter.
xDrip or Nightscout sync could be theoretically handle this because they are bidirectional.
But such a feature will never be implemented because you can measure the bg value only if you have the collecting device.
